components/02-elements/swatch/_swatch.scss

```diff
@@ -0,0 +1,86 @@
+$swatch-font-size        : $font-size-small !default;
+$swatch-margin           : 15px !default;
+$swatch-width            : 48px !default;
+$swatch-transition       : $transition-base !default;
+
+$swatch-background       : $white !default;
+$swatch-color            : $gray-light !default;
+$swatch-border-color     : $swatch-color !default;
+
+$swatch-color-hover      : $color-primary !default;
+
+$swatch-background-active: $color-primary !default;
+$swatch-border-active    : $color-primary !default;
+$swatch-color-active     : $white !default;
+
```

Comment:
Do these tactical holes follows any methodology describing how to place them?

Reply:
Just to info where one section is ending and where next is starting. You will not lost in it i think. Active statement have own group of variables, hover own and normal own too. You see everything without too much searching. Can just delete it.

Reply:
The problem is that it's not logically divided or ordered

Reply:
🆗